lib/mocha.js

```diff
@@ -9,6 +9,7 @@
 var escapeRe = require('escape-string-regexp');
 var path = require('path');
 var builtinReporters = require('./reporters');
+var growl = require('./growl');
```

Comment:
should we be telling the browser to stub this file?

Reply:
Don't think so. Part of Mocha's public API previously. This continues same (with warning/error).

Reply:
I would prefer to leave support as an option for both environments. Although not yet available in IE, it is possible to provide similar notifications within browser using the [Notification API](https://developer.mozilla.org/en-US/docs/Web/API/notification). But it would need to come in a different PR to allow time for sufficient testing.

Reply:
Or not?